osu.Game.Rulesets.Rush/Judgements/SawbladeJudgement.cs

```diff
@@ -1,10 +1,17 @@
 // Copyright (c) Shane Woolcock. Licensed under the MIT Licence.
 // See the LICENCE file in the repository root for full licence text.
 
+using osu.Game.Rulesets.Scoring;
+
 namespace osu.Game.Rulesets.Rush.Judgements
 {
     public class SawbladeJudgement : RushJudgement
     {
-        public override double HealthPoints => -20;
+        protected override double HealthPointIncreaseFor(HitResult result, bool playerCollided) =>
+            result switch
+            {
+                HitResult.Miss => -20.0,
```

Comment:
Like hearts, sawblades are not time-based judgement and should do damage if the player was within collision range during the trailing `Miss` hit window.

Reply:
oh so `playerCollided => -20.0`? sounds good.

Reply:
Depends when the collision check is done.  Sawblades should never be user triggered, and should count as "ouch" if the player was in collision range at any point in the trailing `Miss` hit window.

Reply:
Let's go with `playerCollided` for now, as I'm seeing the sawblade judges are actually based on the collision range.